offixsdk/src/main/java/org/aerogear/offixsdk/provider/Provider.kt

```diff
@@ -0,0 +1,35 @@
+package org.aerogear.offixsdk.provider
+
+import android.content.ContentProvider
+import android.content.ContentValues
+import android.database.Cursor
+import android.net.Uri
+import android.util.Log
+import org.aerogear.offixsdk.isNetwork
+
+internal class Provider : ContentProvider() {
+
+    private val TAG: String = javaClass.simpleName
+
+    override fun insert(uri: Uri, values: ContentValues?): Uri? = null
+
+    override fun query(
+        uri: Uri,
+        projection: Array<String>?,
+        selection: String?,
+        selectionArgs: Array<String>?,
+        sortOrder: String?
+    ): Cursor? = null
+
+    override fun onCreate(): Boolean {
+        Log.e(TAG, " NetStatus in Provider: ${isNetwork(context)}")
+        return isNetwork(context)
```

Comment:
Why are you returning the result of `isNetwork()` here?
Check the docs once to see when you should be returning true/false for this method